heartbeat/monitors/monitor.go

```diff
@@ -35,7 +35,7 @@ import (
 )
 
 // ErrMonitorDisabled is returned when the monitor plugin is marked as disabled.
-var ErrMonitorDisabled = fmt.Errorf("monitor not loaded, plugin is disabled")
+var ErrMonitorDisabled = fmt.Errorf("monitor not loaded, disabled with enabled: false")
```

Comment:
Maybe i misunderstood, We wanted to not throw error and instead log an `info` level message instead with monitor being disabled. I believe the PR does not resolve the issue?

Reply:
Yeah I chatted with @andrewvc about this offline and the primary goal is to ensure this isn't logged as an error. Since the message shows up in the log as `info` level, we determined we didn't need to modify the actual `fmt` call, just the message copy.

Reply:
Gotcha, Thanks for the details Justin.

Reply:
No prob, sorry I didn't post an explanation to the PR.